etc/builtin.conf

```diff
@@ -46,18 +46,27 @@ vlang=
 alang=
 slang=
 
-[gpu-hq]
-scale=spline36
-cscale=spline36
-dscale=mitchell
-dither-depth=auto
+[fast]
+scale=bilinear
+dscale=bilinear
+dither=no
+correct-downscaling=no
+linear-downscaling=no
+sigmoid-upscaling=no
+hdr-compute-peak=no
+
+[high-quality]
+vo=gpu-next
+scale=ewa_lanczossharp
 hdr-peak-percentile=99.995
 hdr-contrast-recovery=0.30
 allow-delayed-peak-detect=no
-correct-downscaling=yes
-linear-downscaling=yes
-sigmoid-upscaling=yes
 deband=yes
+scaler-lut-size=8
+
+# Deprecated alias
+[gpu-hq]
+profile=high-quality
```

Comment:
I think we should follow here principle of least surprise and not suddenly upgrade all users who have `profile=gpu-hq` somewhere in their configs to `vo=gpu-next`.

At least, not yet. For the time being, and given that this profile is now mostly redundant with defaults, we can just make it imply `deband=yes` and maybe also the HDR settings. (Although those were added recently and have non-negligible performance hit..)

Reply:
> I think we should follow here principle of least surprise and not suddenly upgrade all users who have profile=gpu-hq somewhere in their configs to vo=gpu-next.

This PR is nothing like "least surprise", defaults are a huge upgrade. I want to shake up things a little. And `vo=gpu-next` needs some adaptation at some point. It will never be easy to change it.

If we want to preserve old behavior. We can shift everything one step up.

Make `fast` profile defaults. Keep old mpv defaults.
Make new defaults `gpu-hq`/`high-quality`.
Make `high-quality` profile `placebo`. 

But I don't like the middle ground of this solution. I think all the changes are trivially revertible for users. Is it annoying, yes. Can we avoid breaking changes forever, not really. Unless you want to support `vo=gpu` forever.

Reply:
To be clear, my conflict is with the intersection of 1. `profile=high-quality implies vo=gpu-next`, and 2. `profile=gpu-hq implies profile=high-quality`. I think `vo=gpu-next` is more breaking/scary change than changing some meaningless defaults. The former replaces the entire renderer semantics, the latter just drops out one scaling routine for another.

But I'm not heavily invested in this either way.

Reply:
what happens on a build without gpu-next? Error with no fallback?

Reply:
Fixed.

I will try to enable gpu-next by default in 5 years from now in this case.